test/hotspot/jtreg/runtime/Safepoint/TestAbortOnVMOperationTimeout.java

```diff
@@ -36,12 +36,16 @@
 
 public class TestAbortOnVMOperationTimeout {
 
+    // A static array is unlikely to be optimised away by the JIT.
+    static Object[] arr = new Object[10_000_000];
```

Comment:
Initialization should be under `args.length > 0` check. We don't need it in the driver code.